### PR TITLE
Drop ProjectLogEntry

### PR DESCRIPTION
**Pull request by hennevogel**
This feature leads to severe problems when there are many Events.
The ProjectLogRotate job will never finish and it's locking
collides with the SendEventEmails Job.

Drop this for now. We will integrate this with notifications...

### CODE REVIEW

File: src/api/app/models/package.rb [hunk 1]
```diff
@@ -1504,7 +1504,7 @@ def update_activity
 #
 # Foreign Keys
 #
-#  fk_rails_9a47aff19d  (kiwi_image_id => kiwi_images.id)
-#  packages_ibfk_3      (develpackage_id => packages.id)
-#  packages_ibfk_4      (project_id => projects.id)
+#  fk_rails_...     (kiwi_image_id => kiwi_images.id)
```
Comment (ChrisBr) [inline src/api/app/models/package.rb]: This is strange, I also get it, however it is unrelated to this PR and changes

File: src/api/app/models/review.rb [hunk 2]
```diff
@@ -269,6 +269,6 @@ def validate_reviewer_fields
 #
 # Foreign Keys
 #
-#  fk_rails_813a4fb24f  (review_id => reviews.id)
-#  reviews_ibfk_1       (bs_request_id => bs_requests.id)
+#  fk_rails_...    (review_id => reviews.id)
```
Comment (ChrisBr) [inline src/api/app/models/review.rb]: Same here

File: src/api/app/models/package.rb (hunk 1 above)
Comment (mschnitzer) [inline src/api/app/models/package.rb]: Yup, I agree with Chris.

File: src/api/app/controllers/webui/feeds_controller.rb [hunk 3]
```diff
@@ -13,31 +12,4 @@ def latest_updates
...
-                rescue
-                  nil
-                end)
-    end
-    @start ||= 7.days.ago
-    @finish = nil
-    unless params[:ending_at].blank?
-      @finish = (begin
-                   Time.zone.parse(params[:ending_at])
-                 rescue
```
Comment (evanrolfe) [inline src/api/app/controllers/webui/feeds_controller.rb]: good riddance!

Comment (hennevogel) [inline src/api/app/controllers/webui/feeds_controller.rb]: ..to bad rubbish

**Comment by ChrisBr** [comment]
Please rebase @hennevogel

**Comment by hennevogel** [comment]
@ChrisBr please if you add the MP label, make a comment why.

**Comment by ChrisBr** [comment]
Added the ``monkey_patch`` tag despite I did NOT apply the whole patch to build.o.o. However, I needed to comment 
https://github.com/openSUSE/open-build-service/blob/master/src/api/app/jobs/send_event_emails.rb#L19

to make build.o.o work again after receiving MANY 

```
Lock wait timeout exceeded; try restarting transaction
```

and wanted to make sure that this does not happen again after the next deployment!

**Comment by hennevogel** [comment]
DO NOT MERGE:

```
11:40  <lnussel> looks like the rss feed for latest_commits got broken!?
11:40  <lnussel> curl https://build.opensuse.org/project/latest_commits/openSUSE:Leap:42.3
11:40  <lnussel> same for sle12, 42.2:Update etc
11:40  <lnussel> iow breaks update crawler
11:41  <henne> lnussel: what is the update crawler?
11:41  <henne> lnussel: this feed is about the be removed...
11:41  <lnussel>  henne: a script that fetches e.g sle packages and submits them to leap
11:41  <lnussel>  henne: there are many scripts relying on this feed to avoid having to crawl all packages all the time
```

**Comment by lnussel** [comment]
please unbreak the feed. leap 42.3 development relies on this

**Comment by hennevogel** [comment]
@adrianschroeter I guess it's save to say that you want us to solve this right? 😜

**Comment by hennevogel** [comment]
I've removed the MonkeyPatch from production

**Comment by evanrolfe** [comment]
Closing this PR. We are fixing ProjectLogRotate in this new PR: https://github.com/openSUSE/open-build-service/pull/3350

**Pull request closed without merging**